pages/docs/guides/validate.md

```diff
@@ -0,0 +1,138 @@
+---
+title: "Validate AsyncAPI documents"
+description: In this guide, you'll learn multiple ways to validate AsyncAPI documents.
+weight: 120
+---
+
+## Introduction
+This guide teaches multiple ways to validate AsyncAPI documents. 
+
+## Create AsyncAPI document
+To validate an AsyncAPI document, you must first create one. You can create one using the [AsyncAPI CLI](https://github.com/asyncapi/cli#installation).
+
+Generate a sample `asyncapi.yaml` file (AsyncAPI document) with the following CLI command: 
+
+```
+asyncapi new --example=default-example.yaml --no-tty
+```
+
+## Validate AsyncAPI documents
+Validating an AsyncAPI document can mean one of two things: 
+- Validation against the specification.
+- Validation against the best practices or company governance rules also known as linting.
+
+### Validate against specification
+Validating against the specification ensures that every content of the document is written in accordance with the AsyncAPI specification. Several tool options exist for validating against the specification: _AsyncAPI Studio_, _AsyncAPI CLI_, and _Parsers_.
+
+#### AsyncAPI Studio validation 
+[AsyncAPI Studio](https://github.com/asyncapi/studio#readme) provides a visual and easy way to validate your AsyncAPI documents against the specification. (It uses the [AsyncAPI JavaScript parser](https://github.com/asyncapi/parser-js) behind the scenes to perform syntax checks and validate documents.)
```

Comment:
```suggestion
[AsyncAPI Studio](https://studio.asyncapi.com/) provides a visual and easy way to validate your AsyncAPI documents against the specification. (It uses the [AsyncAPI JavaScript parser](https://github.com/asyncapi/parser-js) behind the scenes to perform syntax checks and validate documents.)
```

Reply:
is it correct to have text in `()` as standalone sentence?

Reply:
I once saw @alequetzalli do this, so I went ahead to read about it. 

This is an example reference.

<img width="545" alt="Capture" src="https://user-images.githubusercontent.com/52763841/203899836-5b6b47c4-5319-4a15-8999-6496c26902be.PNG">